frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -131,8 +131,12 @@ action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.title=Git
 action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.delete-text=Slide Out \\& &Delete Local Branch
 action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.cancel-text=&Cancel (do not slide out)
 action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.keep-text=Slide Out \\& &Keep Local Branch
-action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.note-1=Git Machete is able to delete the local branches after sliding them out.
-action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.note-2=\
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.note-1=\
+<html> \
+  You are going to slide out branch <b>{0}</b>.\
+</html>
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.note-2=Git Machete is able to also delete the local branches after sliding them out.
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.note-3=\
 <html> \
   You can change it by setting the git config <b>{0}</b> to <b>true</b> or <b>false</b>. \
```

Comment:
Hmmm actually probably a pre-existing problem with the trailing text being cropped:

![image](https://user-images.githubusercontent.com/3383210/155752668-08c33fa7-4e0c-43fc-b542-91aa2fa16315.png)

Reply:
(`false` missing 🤔)

Reply:
And responding to:

> Also one more thought. Would it be beneficial to add some information on this dialog, that Remember my choice is not some Intellij default behaviour, but rather our own addition, which under the hood adds git config entry like this: machete.slideOut.deleteLocalBranch = true?

Hmm it is kind of suggested by _You can change it..._, but I agree that's not 100% unambiguous 🤔

Reply:
Sorry for slow response. I will check today why this part is missing. Which resolution did you use?

Reply:
1920x1080

Reply:
np! 🇳🇵

Reply:
FYI, changed to sth like this:

![image](https://user-images.githubusercontent.com/3383210/160404341-3704d607-5d9e-480e-aafa-ac09414a9ed5.png)